vms/platformvm/block/serialization_test.go

```diff
@@ -142,20 +145,23 @@ func TestBanffProposalBlockJSON(t *testing.T) {
 	simpleBanffProposalBlockBytes, err := json.MarshalIndent(simpleBanffProposalBlock, "", "\t")
 	require.NoError(err)
 
-	require.Equal(`{
+	expectedSimpleBanffProposalBlockString := `{
```

Comment:
This would be clearer if it used `fmt.Sprintf()` instead of `strings.Replace()`.

(optional, subjective) You could use [explicit arg indices](https://pkg.go.dev/fmt#hdr-Explicit_argument_indexes) if it makes it more readable, but I'm not sure that it will.

Reply:
TIL about explicit arg indicies